packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -48,7 +50,7 @@ function getMessages(results: Array<TestResult> | undefined) {
       .filter((m): m is RegExpExecArray => m !== null)
       .map(
         ([message, line, col]) =>
-          `::error file=${testFilePath},line=${line},col=${col}::${message}`,
+          `\n::error file=${testFilePath},line=${line},col=${col}::${message}`,
```

Comment:
Is there a better solutions? Seems like a conflict between reporters. Before this fix:

<img width="1019" alt="Screenshot 2022-04-09 at 20 05 29" src="https://user-images.githubusercontent.com/72159681/162584482-c1150174-976c-4ade-8261-2c75f9099184.png">

Reply:
A newline seems sensible